### PR TITLE
RIP-7759: L2 Transaction Fee specification

### PR DESCRIPTION
**Pull request by Therecanbeonlyone1969**

### CODE REVIEW

File: RIPS/rip-XXXX.md [hunk 1]
```diff
@@ -0,0 +1,970 @@
+---
+rip: XXX
```
Comment (abcoathup) [inline RIPS/rip-XXXX.md:2]:
```suggestion
rip: 7759
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

File: RIPS/rip-XXXX.md [hunk 2]
```diff
@@ -0,0 +1,970 @@
...
+title: Layer 2 Transaction Fee Specification
```
Comment (abcoathup) [inline RIPS/rip-XXXX.md:3]:
```suggestion
title: Layer 2 Transaction Fees
```
Specification seems superfluous in a RIP title.

File: RIPS/rip-XXXX.md [hunk 3]
```diff
@@ -0,0 +1,970 @@
...
+description: Specification of rules for L2 transaction fee derivation, payout, and display to any of the participants in an L2 platform.
+author: Kelvin Fichter (@smartcontracts), Andreas Freund (@Therecanbeonlyone1969), Daniel Goldman (@DZGoldman), Nikolai Prokhorenko (@0x000hx)
+discussions-to: https://ethereum-magicians.org/t/rip-xxxx-layer-2-transaction-fee-specification/20862
```
Comment (abcoathup) [inline RIPS/rip-XXXX.md:6]:
```suggestion
discussions-to: https://ethereum-magicians.org/t/rip-7759-layer-2-transaction-fee-specification/20862
```
Added RIP number to Eth Magicians topic title.

File: RIPS/rip-XXXX.md (hunk 1 above)
Comment (Therecanbeonlyone1969) [reply RIPS/rip-XXXX.md]: done

File: RIPS/rip-XXXX.md (hunk 3 above)
Comment (Therecanbeonlyone1969) [reply RIPS/rip-XXXX.md]: updated

File: RIPS/rip-XXXX.md (hunk 2 above)
Comment (Therecanbeonlyone1969) [reply RIPS/rip-XXXX.md]: updated

**Comment by Therecanbeonlyone1969** [comment]
@abcoathup Is there a reason for not merging this PR?

**Comment by abcoathup** [comment]
@Therecanbeonlyone1969 you need a RIP editor to review (I am not an editor).  

You could try asking in the RollCall channel of Eth R&D Discord: https://discord.com/channels/595666850260713488/829412277782315008

**Comment by Therecanbeonlyone1969** [comment]
> @Therecanbeonlyone1969 you need a RIP editor to review (I am not an editor).
> 
> You could try asking in the RollCall channel of Eth R&D Discord: https://discord.com/channels/595666850260713488/829412277782315008

@abcoathup thank you! Will do

**Review by CarlBeek — APPROVED** [review]

**Pull request merged**